governance/votes/templates/exec vote ex2.md

```diff
@@ -0,0 +1,128 @@
+
+The Governance Facilitators, Dewiz, Sidestream, and Phoenix Labs have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes** will occur and **additions** will be made to the Maker Protocol:
+- The Enhanced Dai Savings Rate (EDSR) will be activated, as detailed below
+- The Spark Protocol Dai Direct Deposit Module Maximum Debt Ceiling will be increased, as detailed below
+- The Harbor Trade Credit (RWA004-A) vault will be changed, as detailed below
+- The New Silver (RWA002-A) Doc will be updated, as detailed below
+- A sum of 134.1 MKR will be distributed to 10 AVC members, as detailed below.
+- A DAO Resolution hash for Monetalis Clydesdale (RWA007-A) will be approved, as detailed below.
+- 2,000,000 DAI from the Launch Project budget will be distributed, as detailed below.
+- A Spark Protocol proxy spell will be triggered, as detailed below.
+
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the above changes and additions are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### Enhanced Dai Savings Rate (EDSR) activation
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmcTRPLx) the DSR will be updated if this executive proposal passes.
+- DSR will be increased by 4.81% from 3.19% to 8%.
+
+
+### Spark Protocol Debt Ceiling Increase
+Per this successful governance [poll](https://vote.makerdao.com/polling/QmSLj3HS), the Spark Protocol Dai Direct Deposit Module (D3M) will be updated if this executive proposal passes.
+
+- The Spark Protocol D3M Maximum Debt Ceiling will be increased by 180 million DAI from 20 million DAI to 200 million DAI.
+
+### Vault Changes
+
+#### Harbor Trade Credit (RWA004-A) Changes
+As per this successful governance [poll](https://vote.makerdao.com/polling/QmR8cYb1), the following changes to the Harbor Trade Credit (RWA004-A) vault will be made if this proposal passes.
+- Decrease the RWA004-A Debt Ceiling by 7 million DAI from 7 million DAI to 0 DAI.
```

Comment:
I know Patrick does this as well, but I don't like it. The phrasing: 'reduce by x from x to y' is bad in this case, and shouldn't be used. It's harder to parse than just saying: 'decrease from 7 million to zero'.

As a general rule I prefer 'zero DAI' to 0 DAI as well. It makes it 100% clear that its intentionally being set to zero, and isn't a typo or unfilled parameter in the copy.